app/build.gradle

```diff
@@ -143,6 +148,7 @@ dependencies {
     implementation "org.ccci.gto.android:gto-support-sync:${deps.gtoSupport}"
     implementation "org.ccci.gto.android:gto-support-util:${deps.gtoSupport}"
 
+    implementation "com.appsee:appsee-android:+"
```

Comment:
I know documentation tends to suggest using wildcard versions, but we will want an explicit version specified. It helps prevent unexpected changes in behavior when a new version of the library is released with an unexpected change

Reply:
I knew this was going to be a soar spot.  It wasn't apparently clear what the release version was but I will spend sometime today to see if I can find one.

Reply:
you can look to see what version it ended up pulling in. Expand the "External Libraries" item in your project file list

Reply:
I found it.